### ISSUE TITLE
Tooltip undefined during $destroy

### ISSUE THREAD

**Issue by deel77** [lumapps/lumX#291]
I am having problems with lx-tooltip, when I use it with dynamically changing table data that somehow it gets undefined earlier than $destroy event is called.
So I get error 
```
TypeError: Cannot read property 'remove' of undefined
    at lumx.js:3178
    ...
```

I would recommend just adding check in the tooltip $destroy handler function like this:
```javascript
$scope.$on('$destroy', function(scope)
        {
            if (tooltip) {
                tooltip.remove();
            }
        });
```

**Comment by lfallet** [comment]
I'm experiencing the same issue.
Using a ng-repeat filtered table whose cells have tooltips, refreshing the table's data throws the error for all hidden lines.

**Comment by clementprevot** [comment]
Will be fixed by PR #339
